extras/help_generator/parse_help.py

```diff
@@ -135,3 +135,4 @@
 
 shutil.rmtree(install_dir, ignore_errors=True)
 shutil.copytree(helpdir, install_dir)
+shutil.rmtree(helpdir, ignore_errors=True)
```

Comment:
Wouldn't it be more efficient to use `shutil.move()` instead of `copytree()` followed by `rmtree()`?